sdk/streamanalytics/Azure.ResourceManager.StreamAnalytics/src/autorest.md

```diff
@@ -158,6 +177,15 @@ directive:
   where: $.definitions
   transform: >
     $.ResourceTestStatus.properties.error['x-ms-client-flatten'] = true;
+- from: outputs.json
+  where: $.definitions
+  transform: >
+    $.PostgreSQLDataSourceProperties.properties.maxWriterCount['type'] = 'integer';
+    $.AzureSqlDatabaseDataSourceProperties.properties.maxWriterCount['type'] = 'integer';
+    $.AzureFunctionOutputDataSourceProperties.properties.maxBatchSize['type'] = 'integer';
+    $.AzureFunctionOutputDataSourceProperties.properties.maxBatchCount['type'] = 'integer';
+    $.AzureSqlDatabaseDataSourceProperties.properties.maxBatchCount['type'] = 'integer';
+    $.ServiceBusQueueOutputDataSourceProperties.properties.systemPropertyColumns['additionalProperties'] = { 'type': 'string' };
```

Comment:
Why do we need to change this type?

Reply:
From this comment: https://apiview.dev/Assemblies/Conversation/f6ca25aaa01143589fbfdef756473779#482d8b487f7a4ad4a4f0290c8d0bb133